documents/questions/understand-git-commands-questions.md

```diff
@@ -5,20 +5,62 @@ Complete the questions below, by editing this file. Please commit often, to your
 You may reference the videos, the cheatsheet or even google to complete these questions. You can even use this project to play with these git commands.
 
 1. Does a commit connect to the remote?
-1. Does a push connect to the remote?
-1. What is a git repo even?
-1. How do you stage a file, and what does it mean?
-1. How do you unstage a file?
-1. What is the output of `git log`?
-1. How is a git repo initialised?
-1. What information will `git status` give you?
-1. What does a git fetch do?
-1. A `git pull` can basically be broken down into these git commands:
-1. What is a rebase?
-1. How do you view information about the remote config of a git repo?
-1. What commands will create a branch called `qa` from `master`
-1. Lets say you've made changes which you want to discard, and you have not yet commited them anyway, what command can you use?
-1. What is the difference between `git merge` and `git merge <branch>`
-1. What is the git command to change a branch's name locally and remotely? 
-1. If you commit a file, is it commited to your current branch or to master?
-1. If you commited a change, but it was for the wrong branch, how can you move the commit?
+No, it just means that Git will start tracking changes made on that file. Changes made will be save to the Local copy of Git.
+
+2. Does a push connect to the remote?
+Yes, a push connects to remote. Ensure that the location of where the changes will be pushed to should be stated. i.e. git push origin master.
+
+3. What is a git repo even?
+A git repo is version control system where by each commit is saved. When needed changed can be reverted to show a previous version whereby the code was in a good state.
```

Comment:
I would say that it is basically a folder, that has been initialised with git, so that git can record any changes within that folder.